cli/Sources/TuistCore/MetadataProviders/XCFrameworkSignatureProvider.swift

```diff
@@ -52,7 +52,7 @@ public struct XCFrameworkSignatureProvider {
     }
 
     private static let signedWithAppleCertificateString = "Authority=Apple Root CA"
-    private static let teamNameRegExPattern = #"Authority=[^:]+?:\s*([^()]+)\s*\(([A-Z0-9]+)\)"#
+    private static let teamNameRegExPattern = #"Authority=[^:]+?:\s*([^()|^\n]+)\s*(\(([A-Z0-9]+)\))?"#
```

Comment:
I'd add a coment above this with some examples of strings that yield a successful match.